src/api/entity/requests.rs

```diff
@@ -0,0 +1,16 @@
+use super::responses::CreateEntityResponse;
+use rustify_derive::Endpoint;
+use std::fmt::Debug;
+
+#[derive(Builder, Debug, Default, Endpoint)]
+#[endpoint(
+    path = "identity/entity",
+    response = "CreateEntityResponse",
+    method = "POST",
+    builder = "true"
+)]
+#[builder(setter(into), default)]
+pub struct CreateEntityRequest {
+    pub name: String,
+    pub policies: String,
+}
```

Comment:
We should add a full feature set of the API documentation for the endpoint.
```suggestion
pub struct CreateEntityRequest {
    pub name: String,
    pub id: Option<String>,
    pub metadata: Option<HashMap<String, String>>,
    pub policies: Option<Vec<String>>,
    pub disabled: Option<bool>,
}
```